mezzanine/project_template/deploy/gunicorn.conf.py.template

```diff
@@ -1,7 +1,8 @@
 from __future__ import unicode_literals
 import multiprocessing
 
-bind = "127.0.0.1:%(gunicorn_port)s"
-workers = multiprocessing.cpu_count() * 2 + 1
+bind = "unix:%(proj_path)s/gunicorn.sock"
+workers = 2
```

Comment:
Why is this hard-coded? I think the common case would be to leverage the number of cores on the machine. Perhaps we can have it default to that, and overideable via a setting.